freeplane does not work with jdk17, so set jdk11 as the default.
This can be overridden by the upstream-created method still.

--- a/freeplane_framework/script/freeplane.sh
+++ b/freeplane_framework/script/freeplane.sh
@@ -5,6 +5,9 @@
 	set -x
 fi
 
+# set openjdk11 as the default, regardless of xbps-alternatives
+[ -z "$FREEPLANE_JAVA_HOME" ] && FREEPLANE_JAVA_HOME=/usr/lib/jvm/openjdk11
+
 ########## FUNCTIONS DEFINITIONS #######################################
 
 _debug() {
